### ISSUE TITLE
Ban checkout and use "modern" commands switch/restore

### ISSUE THREAD

**Issue by kaddkaka** [pcottle/learnGitBranching#1095]
Has there been any work to replace checkout with the more "direct" commands `switch` and `checkout`? I hesitate to give this learning material to new git users at work because it seems to use checkout.

**Comment by kaddkaka** [comment]
Oh right! On `git 2.42.0` `man git switch` says "THIS COMMAND IS EXPERIMENTAL. THE BEHAVIOR MAY CHANGE."

So it might be good to wait with the porting for that reason. I just forgot about it since I stopped using `checkout` completely since over a year ago.

Anyway, thanks for a really nice interactive learning tool! :)

**Comment by pcottle** [comment]
Yep, it's easy to forget it's experimental. It's like when Google had their products in Beta for years!

I'll probably switch the lessons when they move it out of experimental.

**Comment by Daniel-at-github** [comment]
Is not in experimental now: https://git-scm.com/docs/git-switch/2.51.0

**Comment by pcottle** [comment]
ah well maybe time to switch it over then (no pun intended). My worry is the churn from translations since the content in english will be new, but maybe I can employ LLM translations as a stop-gap...

@kaddkaka do you think thatd make a real difference in recommending the content at your work?
